.github/workflows/build.yml

```diff
@@ -126,7 +126,9 @@ jobs:
         cmake -B build -DCMAKE_C_FLAGS="-Werror" && cmake --build build -j
 
     - name: baresip static
-      if: ${{ matrix.os == 'ubuntu-22.04' }}
       run: |
         rm -Rf build
-        cmake -B build -DSTATIC=1 -DCMAKE_C_FLAGS="-Werror" && cmake --build build -j
+        cmake -B build -DSTATIC=1 -DMODULES="g711;ausine;fakevideo;auconv;auresamp;dtls_srtp;srtp;aufile" && cmake --build build -j
```

Comment:
Just curious, why did you remove `-DCMAKE_C_FLAGS="-Werror"`?

Reply:
I put back the Werror ... no specific reason to remove it ...